packages/studio-ui/src/web/reducers/ui.ts

```diff
@@ -92,6 +93,12 @@ const reducer = handleActions(
         zoomLevel: state.zoomLevel + 25
       }
     },
+    [zoomToFit]: (state, {}) => {
+      return {
+        ...state,
+        zoomLevel: -1
```

Comment:
Can you put this into a reusable constant so we know that this magic number means that the zoom should be adjusted?